starter-code/index.html

```diff
@@ -1,42 +1,103 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
   <meta charset="UTF-8">
   <title>Spotify</title>
+  <link rel="stylesheet" type="text/css" href="css/main.css">
 </head>
+
 <body>
-Premium Discover Help Download
-
-Music for everyone.
-Spotify is now free on mobile, tablet and computer.
-Listen to the right music, wherever you are.
-
-What’s on Spotify?
-
-Millions of Songs
-There are millions of
-songs on Spotify
-
-HD Music
-Listen to music as if you
-were listening live
-
-Stream Everywhere
-Stream music on your
-smartphone, tablet or
-computer
-
-It’s as yeezy as Kanye West.
-Search
-Know what you want to listen to?
-Just search and hit play.
-Browse
-Check out the latest charts,
-brand new releases and great
-playlists for right now.
-Discover
-Enjoy new music every Monday
-with your own personal playlist.
-Or sit back and enjoy Radio.
+  <div>
+    <nav>
+      <section class="spotify-logo">
+        <img class="spotify-logo" src="./images/spotify-logo.png" alt="spotify-logo">
+      </section>
+      <section class="links">
+        <a href="#">Premium</a>
+        <a href="#">Discover</a>
+        <a href="#">Help</a>
+        <a href="#">Download</a>
+      </section>
+    </nav>
+  </div>
+
+  <div class="container">
+    <img src="./images/landing.jpg" alt="landing">
+    <div class="header">
+      <h1>Music for everyone.</h1>
+      Spotify is now free on mobile, tablet and computer.
+      <br> Listen to the right music, wherever you are.
+    </div>
+  </div>
+
+  <div>
+    <section>
+      <h2 class="what-spotify">What’s on Spotify?</h2>
+      <hr class="green-line">
+    </section>
+
+    <section class="middle">
+      <div class="box">
+        <img class="middle-images" src="./images/music-icon.png" alt="music-icon">
+        <section class="text">
```

Comment:
I would change all these inside section tags with div tags, just because section tag groups part of a page that logically belongs together so making wrapper with section tag makes sense but having a lot of inside section tags really doesn't. Check this:
https://stackoverflow.com/questions/6939864/what-is-the-difference-between-section-and-div